### PR TITLE
Update Gojo, University of California, and Balkan_irl entries

### PR DESCRIPTION
**Pull request by GeoDash897**

### CODE REVIEW

File: web/atlas.json [hunk 1]
```diff
@@ -3784,7 +3782,7 @@
...
-{"id": "tx40ya", "name": "2balkan4you", "description": "In memory of the subbredit 2balkan4you banned by u/Chtorrr in the beginning 2022, where people from the balkans could joke together.\nMany new subreddits have tried to reconstruct the community of around 200k members, but the majority of them are empty.", "links": {"website": ["https://subredditstats.com/r/2balkan4you"]}, "path": {"56-166, T": [[1918, 322], [1918, 334], [1936, 334], [1936, 322]]}, "center": {"56-166, T": [1927, 328]}},
+{"id": "tx40ya", "name": "2balkan4you", "description": "In memory of r/2balkan4you banned by u/Chtorrr in the beginning 2022, a subreddit where people from the Balkans could joke together.\n\nMany new subreddits have tried to reconstruct the community of around 200k members, but the majority of have become empty.", "links": {"website": ["https://subredditstats.com/r/2balkan4you"], "subreddit": ["balkans_irl"]}, "path": {"56-166, T": [[1918, 322], [1918, 334], [1936, 334], [1936, 322]]}, "center": {"56-166, T": [1927, 328]}},
```
Comment (AnonymousRandomPerson) [inline web/atlas.json:3785]: "majority of **them** have become empty"

Comment (GeoDash897) [reply web/atlas.json:3785]: Implemented fix.

Thanks for the catch!

File: web/atlas.json [hunk 2]
```diff
@@ -5255,7 +5253,7 @@
...
-{"id": "txmlog", "name": "Annoying Dog", "description": "Undertale is an indie role-playing game by Toby Fox, who is represented in the game as a small white dog which usually ruins a character's plan whenever it appears.\n\n*Bark* *Bark*", "links": {"website": ["https://undertale.fandom.com/wiki/Annoying_Dog"], "subreddit": ["Undertale"]}, "path": {"15-164, T": [[671, 763], [681, 763], [685, 766], [690, 766], [690, 775], [688, 779], [682, 779], [681, 778], [679, 778], [678, 779], [671, 779]]}, "center": {"15-164, T": [679, 771]}},
+{"id": "txmlog", "name": "Annoying Dog", "description": "Undertale is an indie role-playing game by Toby Fox, who is represented in the game as a small white dog which usually ruins a character's plan whenever it appears.\n\n*Bark* *Bark*", "links": {"website": ["https://undertale.fandom.com/wiki/Annoying_Dog"], "subreddit": ["Undertale", "deltarune "]}, "path": {"15-164, T": [[671, 763], [681, 763], [685, 766], [690, 766], [690, 775], [688, 779], [682, 779], [681, 778], [679, 778], [678, 779], [671, 779]]}, "center": {"15-164, T": [679, 771]}},
```
Comment (AnonymousRandomPerson) [inline web/atlas.json:5256]: Extra space after `deltarune`.

Comment (GeoDash897) [reply web/atlas.json:5256]: Fixed

**Pull request merged**